app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/supportingevidence/SupportingEvidenceController.scala

```diff
@@ -447,10 +447,12 @@ object SupportingEvidenceController {
     supportingEvidenceDocumentType: SupportingEvidenceDocumentType
   )
 
+  val chooseDocumentTypeDataKey = "supporting-evidence.choose-document-type"
+
   val chooseSupportEvidenceDocumentTypeForm: Form[ChooseSupportingEvidenceDocumentType] =
     Form(
       mapping(
-        "supporting-evidence.choose-document-type" -> number
+        chooseDocumentTypeDataKey -> number
```

Comment:
Looks like somebody has removed some document type options here: 

```scala
     documentType === 0 ||
                documentType === 1 ||
                documentType === 2 ||
                documentType === 3 ||
                documentType === 4 ||
                documentType === 5 ||
                documentType === 7 ||
                documentType === 8 ||
                documentType === 10 ||
                documentType === 11
```

i.e. it goes from 0 to 8 and then jumps to 10. It would be good to get that back in order. You will have to modify the Twirl template accordingly as well.

Reply:
6 is missing, too. Shall we have it?